src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -171,6 +176,12 @@ export default {
                 return this.itemIsInCurrentSelection(item);
             });
         },
+        layoutDimensions() {
+            return this.internalDomainObject.configuration.layoutDimensions;
+        },
+        layoutDimensionsStyle() {
+            return `width: ${this.layoutDimensions[0]}px; height: ${this.layoutDimensions[1]}px;`;
```

Comment:
Should this return an object rather than a string?
Elsewhere in the Open MCT code computed styles are returned as objects.

Reply:
returning an object will be consistent with the way styles are applied else where.

Reply:
@shefalijoshi , @deeptailor , thanks. done.